src/main/java/org/apache/mesos/config/RepairConfiguration.java

```diff
@@ -0,0 +1,93 @@
+package org.apache.mesos.config;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Represents the repair configuration in JSON.
+ */
+public class RepairConfiguration {
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+        return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+
+        RepairConfiguration that = (RepairConfiguration) o;
+
+        if (gracePeriodMins != that.gracePeriodMins) {
+            return false;
+        }
+        if (repairDelaySecs != that.repairDelaySecs) {
+            return false;
+        }
+        return enableReplacement == that.enableReplacement;
+
+    }
+
+    @Override
+    public int hashCode() {
+        int result = gracePeriodMins;
+        result = 31 * result + repairDelaySecs;
+        result = 31 * result + (enableReplacement ? 1 : 0);
+        return result;
+    }
+
+    @JsonProperty("repairInPlaceGracePeriodMins")
+    private int gracePeriodMins;
+    @JsonProperty("minDelayBetweenRepairsSecs")
+    private int repairDelaySecs;
+    @JsonProperty("enableReplacement")
+    private boolean enableReplacement;
+
+    public  RepairConfiguration() {}
+
+    @JsonCreator
+    public RepairConfiguration(
+            @JsonProperty("repairInPlaceGracePeriodMins") int gracePeriodMins,
+            @JsonProperty("minDelayBetweenRepairsSecs") int repairDelaySecs,
+            @JsonProperty("enableReplacement") boolean enableReplacement) {
+        this.gracePeriodMins = gracePeriodMins;
+        this.repairDelaySecs = repairDelaySecs;
+        this.enableReplacement = enableReplacement;
+    }
+
+    public boolean isEnableReplacement() {
+        return enableReplacement;
+    }
+
+    @JsonProperty("enableReplacement")
+    public void setEnableReplacement(boolean enableReplacement) {
+        this.enableReplacement = enableReplacement;
+    }
+
+    public int getRepairDelaySecs() {
```

Comment:
Given that JSON doesn't have a duration type, I didn't want to add custom serialization/deserialization code, since this object is a json data model. Do you require this changed? If so, do you want me to use java.time or joda.time Durations? And do you mind if I continue to document the duration in the method, since at the usage sites I need to convert to other APIs?